core/src/main/java/io/github/tcdl/msb/api/ResponderContext.java

```diff
@@ -0,0 +1,25 @@
+package io.github.tcdl.msb.api;
+
+import io.github.tcdl.msb.api.message.Message;
+
+/**
+ * Provides access to Responder Context.
+ */
+public interface ResponderContext {
```

Comment:
I don't get we introduced `ResponderContext` it seems to be not symmetrical to what we have for `Requester` where we pass ack handler explicitly.